printing/printer.ml

```diff
@@ -289,10 +289,8 @@ let pr_global_env = pr_global_env
 let pr_global = pr_global_env Id.Set.empty
 
 let pr_puniverses f env (c,u) =
-  f env c ++ 
-  (if !Constrextern.print_universes then
-    str"(*" ++ Univ.Instance.pr Universes.pr_with_global_universes u ++ str"*)"
-   else mt ())
+  if !Constrextern.print_universes then f env c else
+    Termops.pr_puniverses (f env c) u
```

Comment:
Isn't this logic inverted? In fact the functions which use it are the the `pr_p{constant,inductive,constructor}` in this file which don't appear anywhere (including API) and after this PR the ill_formed_branch error explainer (which I guess we never test with `Universe Polymorphism` or `Printing Universes` since there's no output test change).
In fact since the explainer is using it to print a constructor (`pr_puniverses pr_constructor env ci`) we should be using the `@{...}` printing, not printing in comments.

Reply:
I looked at this again and I couldn't figure out if my comment about the users of this function was right.
The logic is definitely inverted though.

Reply:
Oh right I see it now, it’s backwards.

Reply:
And indeed this is a remnant of when we didn’t print instances it seems.